src/tests/GC/Stress/Framework/ReliabilityFramework.cs

```diff
@@ -157,9 +157,20 @@ public static int Main(string[] args)
         string configFile = null;
         bool okToContinue = true, doReplay = false;
         string sTests = "tests", sSeed = "seed", exectime = "maximumExecutionTime";
+        string output = "";
```

Comment:
Making use of StringBuilder here. https://learn.microsoft.com/en-us/dotnet/standard/base-types/stringbuilder

```csharp
StringBuilder sb = new()
...

// To Add..
sb.AppendLine($"{kvp.Key}: {kvp.Value}");

...
// To Log..
rf._logger.WriteToInstrumentationLog(null, LoggingLevels.StartupShutdown, String.Format("Get GC Configuration Variables\n{0}",output.ToString()));
```

Reply:
or perhaps don't bother with `output` at all and just write the lines one at the time?

Reply:
@mrsharm I have updated the source with using StringBuilder.